include/scheduler_callback.iol

```diff
@@ -0,0 +1,14 @@
+type SchedulerCallBackRequest: void {
```

Comment:
This file has been removed. Please remove it.

Reply:
If we remove it, the programmer must define the inputPort and the interface